### DISCUSSION TITLE
Cruiser 3.0.21 gives an error and closes

### DISCUSSION THREAD

**Discussion by ramondevic** [devemux86/cruiser#196] [Q&A] [answered]
I'm in Iran Kurdistan preparing the routes for tomorrow and when I export it to start a new one it gives an error and closes.

This happens with version 3.0 21, with 3.0.20 it worked perfectly.

Is there any way to go back to the previous version?

<img src="https://github.com/devemux86/cruiser/assets/122691401/edb3029c-c4fb-4496-9ddb-8bee28ef25a9" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/122691401/53906f1f-3867-437c-89f2-f33b3a7d2ad4" width="50%"/>

Thanks !

**Comment by ramondevic** [comment]
An error also occurs when exporting Favorites to work with Kurviger Pro.

**Comment by Ciutnik** [comment]
A moment ago, application 3.0.21 force-closed when trying to load a custom map theme. Previously, this happened multiple times during a map update when the tablet screen went blank.

**Reply by ramondevic** [reply]
Cruiser closes whenever a function outside of Cruiser is executed such as exporting/importing, switching applications, clicking the Home button to go to the phone's desktop.
That is, when Cruiser loses focus to pass it to another application.

**Comment by devemux86** [comment]
Thanks everyone for the report!

Do you have the **Kalman filter** enabled?
Please turn off the **Kalman filter** for now.

I will release a new version with the fix the next hours!

**Reply by devemux86** [reply]
Cruiser 3.0.22 with the fix was released.

**Reply by ramondevic** [reply]
Thanks, it's working again !

**Reply by Ciutnik** [reply]
Thanks! You are very fast and effective! 👍

**Comment by devemux86** [comment, marked as answer]
It should work properly again in [Cruiser 3.0.22](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by Ciutnik** [comment]
BTW: Are crash reports now automatically sent to the developer? In previous versions of Android, I often saw a button like "send report to developer" in such cases. I haven't seen anything like that lately.

**Reply by devemux86** [reply]
I also saw this error report in Google Play console.

It also depends on the timing of an error and if the Android system can display the report dialog.

**Reply by Ciutnik** [reply]
Well, the thing is that no dialog box was displayed. Only some perfunctory information that there was an error and... that I should wait for the developer to release a new version of the application 😃 (this was on a Samsung with Android 13)
